### ISSUE TITLE
Drop pod presets can't be created

### ISSUE THREAD

**Issue by iprice** [tgstation/tgstation#81140]
Reporting client version: 515.1630

## Round ID:
[222930](https://scrubby.melonmesa.com/round/222930)

## Testmerges:
- [Temp keybinds patch for 515.1628](https://www.github.com/tgstation/tgstation/pull/81089)

## Reproduction:

Been a while, I noticed all my saved drop pod configs had gone and cursed quietly and told myself i'd rebuild them all later.  Which I never did.  But given drop pods are pretty long winded to set up for the massive variety of effects they can do, tonight I tried.

In short, open the supply drop pod "Config/Launch-Supplypod" and click the + by presets, put in a name, and a hue if you like and hit confirm.  No preset is created.
